webapp/tests/i18n_mock.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this file really needed?

Reply:
I've included it in all repos where I've introduced tests. A quick test with the file deleted shows that it is not necessary to run the tests. But I have a feeling it will be necessary once we wave i18n set up and have tests using it. It is mentioned in the `jest` config in `package.json` https://github.com/mattermost/mattermost-plugin-github/blob/0e8eeb79d7a92e580380954742569d066d3f23c3/webapp/package.json#L99

Maybe we should pull the `jest` config out of `package.json`, into its own file, to minimize the amount of content shared in `package.json`.

Reply:
Do you think we should remove this file from the jest config and remove it from all plugin repos?